modules/end-user-guide/pages/enabling-maven-artifact-repositories.adoc

```diff
@@ -0,0 +1,108 @@
+:navtitle: Maven
+:keywords: maven, artifact-repository, artifact-repositories
+:page-aliases: .:using-maven-artifact-repositories
+
+[id="enabling-maven-artifact-repositories"]
+= Enabling Maven artifact repositories
+
+To enable a Maven artifact repository in Maven workspaces that run in a restricted environment:
+
+.Prerequisites
+
+* You are not running any Maven workspace.
```

Comment:
Should this be in the form of a directive, like:
* Ensure all maven based workspaces are stopped.

Reply:
`You are not running any workspaces` is a condition, whereas we put directives into the procedure.
It's also possible that the reader has no workspaces that must be stopped.

Reply:
Yes, but this is the end user guide, so the reader is the one performing the procedure and to whom the prerequisite applies.

Reply:
I'll check with Tibor.

Reply:
Resolved on Slack.